modules/nf-core/pindel/pindel/main.nf

```diff
@@ -0,0 +1,57 @@
+process PINDEL_PINDEL {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda (params.enable_conda ? "bioconda::pindel=0.2.5b9" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/pindel:0.2.5b9--h06e5f0a_6':
+        'quay.io/biocontainers/pindel:0.2.5b9--h06e5f0a_6' }"
+
+    input:
+    tuple val(meta), path(bam), path(bai), val(insert)
+    path fasta
+    path fai
+    path bed
+
+    output:
+    tuple val(meta), path("*_BP")            , emit: bp
+    tuple val(meta), path("*_CloseEndMapped"), emit: cem
+    tuple val(meta), path("*_D")             , emit: del
+    tuple val(meta), path("*_DD")            , emit: dd, optional:true
+    tuple val(meta), path("*_INT_final")     , emit: int_final
+    tuple val(meta), path("*_INV")           , emit: inv
+    tuple val(meta), path("*_LI")            , emit: li
+    tuple val(meta), path("*_RP")            , emit: rp
+    tuple val(meta), path("*_SI")            , emit: si
+    tuple val(meta), path("*_TD")            , emit: td
+    path "versions.yml"                      , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    def insert_size = insert ?: "500"
+
+    if (bam instanceof nextflow.util.BlankSeparatedList) {
+        error "pindel/pindel only takes a single bam file as input"
+    }
+
+    """
+    echo -e "${bam}\t${insert_size}\t${prefix}" > pindel.cfg
```

Comment:
On the other hand, maybe using an `ext.args2` here could solve the problem of having a value input. You could document in the `meta.yml` that the user should specify the `insert_size` inside of `ext.args2`. This will be the most flexible we'll get with this input

Reply:
Nope, you can add the different sizes to meta, e.g. `[insert_size:200]`. You can access this value inside of `modules.conf` like this: `ext.args2 = {"${meta.insert_size}"}. This way it will use a different value based on the meta given with the BAM file

Reply:
ah, ok, so you guys want the module to read `insert_size` from `ext.args2` and setting `ext.args2` to `meta.insert_size` happens outside of the module on workflow level?

Reply:
That's correct!! :tada: